routes/alarms.js

```diff
@@ -28,15 +28,16 @@ router.get('/acknowledge', function(req, res) {
 });
 
 router.get('/test', function(req, res) {
-  var now = new Date();
-  var base = '2021-12-20T';
-  if (Date.parse(base+'09:30:00+01:00') < now && now < Date.parse(base+'10:30:00+01:00')) {
-    req.logging_db.get('alarm_history').insert({'msg': 'Test alarm', acknowledged: parseInt('0'), level: parseInt('2')})
-    .then(() => res.send('Ring ring ring...'))
+  //var now = new Date();
+  //var base = '2021-12-20T';
+  //if (Date.parse(base+'09:30:00+01:00') < now && now < Date.parse(base+'10:30:00+01:00')) {
+  var level = req.level;
+    req.logging_db.get('alarm_history').insert({'msg': 'Test alarm', acknowledged: parseInt('0'), level: parseInt(level)})
```

Comment:
This doesn't really work anymore because of changes to how alarms are generated internally in Doberman. We should discuss how to deal with this, it'll probably involve sending something via a socket but we should make sure Doberman is listening properly.